### PR TITLE
Remove web steps

### PR DESCRIPTION
**Pull request by jburel**
Remove ref to web in files related to omero-install

Nothing to test: check by superbuild

### CODE REVIEW

**Comment by jburel** [comment]
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs-superbuild-build/49/ is now green

**Comment by will-moore** [comment]
Looks good 👍

File: omero/sysadmins/unix/install-web.rst [hunk 1]
```diff
@@ -68,7 +68,7 @@ display a maintenance page.
       $ omero web clearsessions
 
   It is recommended to call this command on a regular basis, for example 
-  as a :download:`daily cron job <walkthrough/omero-web-cron>`, see
```
Comment (joshmoore) [inline omero/sysadmins/unix/install-web.rst:71]: This feels like something is being lost.

Comment (joshmoore) [reply omero/sysadmins/unix/install-web.rst:71]: Could the download have been moved to _this_ page?

Comment (jburel) [reply omero/sysadmins/unix/install-web.rst:71]: The example did not add much. I can look at adding them to the doc. 
They should be in omeroweb-install in theory but that will require reviewing the generation of the doc.

**Pull request merged**